images/aquarium/config.xml

```diff
@@ -72,6 +72,8 @@
         <packagemanager>zypper</packagemanager>
         <locale>en_US</locale>
         <keytable>us</keytable>
+        <!-- If you're wondering why there's no manpages included in the
+             image, this next setting is what ensures they're not installed -->
```

Comment:
Maybe I'm just daft, but this is really confusing. "If you're wondering why this thing doesn't thing, the next thing is why that thing doesn't thing"

How about, "The thing doesn't work and this is by design. See below"

Reply:
Maybe I'm just daft, but it sounds perfectly reasonable to me xD